lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -877,6 +880,42 @@ def write(chunk)
       end
     end
 
+    def dynamic_target_index_enabled?()
```

Comment:
`enable_index_affinity?` or something is better.

Reply:
ok